cmd/import-dashboards/main.go

```diff
@@ -66,7 +66,7 @@ func main() {
 				continue
 			}
 			c.DeleteDashboard(board.UpdateSlug())
-			_, err := c.SetDashboard(board, false)
+			_, err := c.SetDashboard(board, 0, false)
```

Comment:
Could we convert `0` into a constant with a name like `const GeneralFolderId = 0` so that it would be clearer what's going on?

Reply:
Okay. I'll change that. Thanks